lib/flame_scrolling_sprite.dart

```diff
@@ -0,0 +1,172 @@
+library flame_scrolling_sprite;
+
+import 'package:flame/sprite.dart';
+import 'package:flame/components/component.dart';
+import 'package:meta/meta.dart';
+import 'dart:ui';
+
+class ScrollingSprite {
+  Sprite _sprite;
+  double _width;
+  double _height;
+  double _spriteWidth;
+  double _spriteHeight;
+  final double verticalSpeed;
+  final double horizontalSpeed;
+  final bool clipToDimensions;
+
+  List<Rect> _chunks = [];
+
+  ScrollingSprite({
+    @required String spritePath,
+    double spriteX = 0.0,
+    double spriteY = 0.0,
+    double spriteWidth,
+    double spriteHeight,
+    double spriteDestWidth,
+    double spriteDestHeight,
+    double width,
+    double height,
```

Comment:
aren't these parameters required as well?

Reply:
Nope, they default to the sprite dimensions in case you don't inform this.